### PR TITLE
TodoList Working

### PR DESCRIPTION
**Pull request by G4GANCHAUDHARY**

### CODE REVIEW

File: assignments/todo_list/todo.html [hunk 1]
```diff
@@ -5,9 +5,9 @@
     <body>
         <div>
             <input type='text' name='todo_input'>
-            <button type='submit'>Add Todo</button>
```
Comment (abhishek-dash) [inline assignments/todo_list/todo.html:8]: You are using two add to do button.why?

**Review by abhishek-dash — CHANGES_REQUESTED** [review]
Testcases are not passing.Please do the necessary changes..

**Comment by abhishek-dash** [comment]
Screenshot (337)

File: assignments/todo_list/todo.html (hunk 1 above)
Comment (G4GANCHAUDHARY) [reply assignments/todo_list/todo.html]: They are not two todo buttons. Its showing one button deleted(-) one button added(+).

**Comment by G4GANCHAUDHARY** [comment]
> Testcases are not passing.Please do the necessary changes..

TBH not checked the testcases earlier cause todo list was working fine...testcases were passing when I removed onclick attribute from button and added a eventlistner to button in script...bizarre test suites!!

**Pull request merged**